module-2/lab-df-calculation-and-transformation/your-code/challenge-2.ipynb

```diff
@@ -13,7 +13,7 @@
   },
```

Comment:
- It's about using apply and lambda to get the three stats, max,min,mean. The way you have defined it doesn't work. 
```Python
min==numeric_cols.apply(lambda x: x.min ())
```
The rest of challenge 2 is ok!!